tests/python_package_test/test_engine.py

```diff
@@ -2042,47 +2046,47 @@ def train_booster(params=params_obj_verbose, **kwargs):
 
     # fobj, feval
     # no default metric, only custom one
-    res = get_cv_result(params=params_verbose, fobj=dummy_obj, feval=constant_metric)
+    res = get_cv_result(params=params_dummy_obj_verbose, feval=constant_metric)
     assert len(res) == 2
     assert 'valid error-mean' in res
 
     # metric in params with custom one
-    res = get_cv_result(params=params_metric_err_verbose, fobj=dummy_obj, feval=constant_metric)
+    res = get_cv_result(params=params_metric_err_verbose, feval=constant_metric)
```

Comment:
Add `dummy_obj` to `params_metric_err_verbose` and rename them into `params_dummy_obj_metric_err_verbose`

Reply:
copy that